### PR TITLE
Barrier

### PR DESCRIPTION
**Pull request by Tunguso4ka**
## About the PR

Ported Deployable barrier from the cm-ss13.

## Media

зображення

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

### CODE REVIEW

**Comment by Skarletto** [comment]
Ain't never seen these in cm13, can you vault over them or do you have to break them?

**Comment by Tunguso4ka** [comment]
> Ain't never seen these in cm13, can you vault over them or do you have to break them?

break i think. I've seen them on one of the station maps.

File: Resources/Prototypes/_CM14/Entities/Structures/Walls/barrier.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+- type: entity
+  parent: DeployableBarrier
+  id: CMDeployableBarrier
+  components:
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_CM14/Entities/Structures/Walls/barrier.yml:4]:
```suggestion
  id: CMDeployableBarrier
  name: deployable barrier
  description: A deployable barrier. Swipe your ID card to lock/unlock it.
  components:
```

File: Resources/Prototypes/_CM14/Entities/Structures/Walls/barrier.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 300
+      behaviors:
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          CMSheetMetal:
+            min: 5
+            max: 5
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_CM14/Entities/Structures/Walls/barrier.yml:35]: As far as I can tell in CM13 these are supposed to have 100 health and drop nothing
```suggestion
    - trigger:
        !type:DamageTrigger
        damage: 100
      behaviors:
        - !type:DoActsBehavior
          acts: [ "Destruction" ]
```

**Review by DrSmugleaf — DISMISSED** [review]

**Comment by Tunguso4ka** [comment]
thank you for fixes.

**Review by QuietlyWhisper — APPROVED** [review]
Thanks

**Pull request merged**